tests/test_statepoint_sourcesep/test_statepoint_sourcesep.py

```diff
@@ -15,6 +15,13 @@ def _test_output_created(self):
         assert source[0].endswith('h5'), \
              'Source file is not a HDF5 file.'
 
+    def _cleanup(self):
+        TestHarness._cleanup(self)
+        output = glob.glob(os.path.join(os.getcwd(), 'source.*'))
+        for f in output:
+            if os.path.exists(f):
+                os.remove(f)
```

Comment:
`os` module is never explicitly imported. It happens to work because `from testing_harness import *` includes `os`, but we shouldn't rely on that.